app/models/transaction.rb

```diff
@@ -14,11 +14,17 @@
 #
 
 class Transaction < ActiveRecord::Base
+  include TimeRange
   AMOUNT_MAX = 21_474_836.47
   TRANSACTION_TYPES = %w(Residue)
-  FILTER_PERIOD = [['Current month', 'current_month'], ['Previous month', 'prev_month'],
-   ['Last 3 months', 'last_3_months'],['Quarter', 'quarter'],
-   ['This year', 'this_year'], ['Custom', 'custom']]
+  FILTER_PERIOD = [
+    ["Current month: #{TimeRange.format(Time.now, 'current')}", "current_month"],
```

Comment:
@Gataniel remove it from here. It will not work correctly. There will be always time of class loading.